modules/user-guide/nav.adoc

```diff
@@ -2,12 +2,14 @@
 
 * xref:introduction.adoc[]
 * xref:installing-vale-cli.adoc[]
-* xref:linting-vale-cli.adoc[]
-* xref:using-vale-in-the-ide.adoc[]
+* xref:analyze-with-vale.adoc[]
+** xref:analyze-vale-app.adoc[]
```

Comment:
@themr0c - 
Does the doc navigation support nested topics? 
If not I will go back to the drawing board on how to restructure the content.